### PR TITLE
Update Moderator & Analyst pages with Queue revamp

### PR DESCRIPTION
**Pull request by natevaut**
Added a page to view the Queue (`/queue`) which includes the status of the article and has an option to remove the article from the queue.

### CODE REVIEW

**Comment by natevaut** [comment]
~~Needs to be integrated into Moderator/analyst page tables?~~
Done

**Comment by natevaut** [comment]
Ready to merge

**Comment by natevaut** [comment]
is set up to merge into 'test' when Merge is clicked

**Comment by natevaut** [comment]
Ready to merge

File: backend/src/models/queuedArticles/dto/create-article.dto.ts [hunk 1]
```diff
@@ -28,4 +28,6 @@ export class CreateArticleDto {
   readonly abstract: string;
   @IsBoolean()
   readonly isModerated: boolean;
+  @IsBoolean()
+  readonly isDuplicate: boolean;
```
Comment (natevaut) [inline backend/src/models/queuedArticles/dto/create-article.dto.ts]: rmv

**Comment by natevaut** [comment]
When testing change DOMAIN.ts to `const DOMAIN = 'http://localhost:3000/';`

File: backend/src/models/queuedArticles/dto/create-article.dto.ts (hunk 1 above)
Comment (janenotjung-hue) [reply backend/src/models/queuedArticles/dto/create-article.dto.ts]: kk

**Pull request merged**